pkg/otlp/logs/logs_translator.go

```diff
@@ -118,6 +118,14 @@ func Transform(lr plog.LogRecord, res pcommon.Resource, logger *zap.Logger) data
 		}
 		return true
 	})
+	res.Attributes().Range(func(k string, v pcommon.Value) bool {
+		if k == "hostname" || k == "service" {
```

Comment:
Maybe add a comment here explaining `hostname` and `service` are reserved keywords in log items and we don't want to override, thus adding the otel. prefix.